cumulus/parachains/runtimes/assets/asset-hub-rococo/src/xcm_config.rs

```diff
@@ -844,7 +860,7 @@ pub mod bridging {
 			/// (initially was calculated by test `OutboundQueue::calculate_fees` - ETH/ROC 1/400 and fee_per_gas 20 GWEI = 2200698000000 + *25%)
 			/// Needs to be more than fee calculated from DefaultFeeConfig FeeConfigRecord in snowbridge:parachain/pallets/outbound-queue/src/lib.rs
 			/// Polkadot uses 10 decimals, Kusama and Rococo 12 decimals.
-			pub const DefaultBridgeHubEthereumBaseFee: Balance = 2_750_872_500_000;
+			pub const DefaultBridgeHubEthereumBaseFee: Balance = 4_200_000_000_000;
```

Comment:
I'd like the `EthereumBasedFee` not a on-chain config here, instead a user input param for the `reserve_transfer_assets` call. 

Meanwhile, there are 2 hops for the fees, the first hop from AH to BH  with fee in DOT and the second hop from BH to Ethereum with fee in Ether.

Reply:
Another option would be to try and extend `NetworkExportTableItem` to include some implementation that allows us to charge a different base fee based on the message. The UX to just use DOT to pay fees is simple. But I do hear you say we can bake in the WETH swap to the user doesn't have to do it.

Reply:
Yeah, that's true. But the main concern here is to remove all the variable configurations which are hard to maintain on chain, including the `ExchangeRate` on BridgeHub and `EthereumBasefee` on Assethub. 

That's why I made the change in https://github.com/Snowfork/polkadot-sdk/pull/135/files#r1549151120

Reply:
I agree! But don't you think if the NetworkExportTable supported calculating the fee with a function instead of just hardcoding a constant, it would be more flexible? We would still have the exchange rate problem but we can improve that with the onchain DEX?

Reply:
You mean based on the user operation? Or what user input?

Reply:
Fees are params of `reserve_transfer_asset`(extrinsic in `pallet_xcm`) from user input, more context in
https://github.com/paritytech/polkadot-sdk/issues/3931#issuecomment-2032086134

Actually, I add a new `reserve_transfer_ethereum_assets` for demonstration in https://github.com/Snowfork/polkadot-sdk/pull/135/files#r1559360864

Reply:
I see, thanks for demonstrating!